### DISCUSSION TITLE
grasp structure from raw/file

### DISCUSSION THREAD

**Discussion by Wangmumu0403** [deepmodeling/deepmd-kit#2006] [Q&A]
I have use all OUTCAR calculated by VASP to get the raw to train NNP, but i forget to store all configuration, so now I want to grasp structure file from the raw file ，does deepmd has any operation to help user to grasp structure from raw file  as a POSCAR or xyz format ?

**Comment by njzjz** [comment]
This is the scope of [dpdata](https://github.com/deepmodeling/dpdata). See [documentation](https://docs.deepmodeling.com/projects/dpdata/).
